client/js/Core/main.js

```diff
@@ -187,16 +197,22 @@ SCWeb.core.Main = {
         function start(a) {
             SCWeb.core.Main.doDefaultCommand([a]);
             if (params.first_time)
-                $('#help-modal').modal({"keyboard": true});
+                $('#help-modal').modal({ "keyboard": true });
+        }
+
+        argumentAddr = window.scKeynodes['ui_start_sc_element'];
+        const last_page_cmd_args = [getCookie('last_page_cmd_args')]
+        if (last_page_cmd_args) {
```

Comment:
This check isn't really correct. `last_page_cmd_args`  contains array with null value in case of empty cookie.
Currently it leads to strange issue - if we open page without cookies then there is no main page content shown